excercise1/AIRA/MyMaths.py

```diff
@@ -0,0 +1,18 @@
+
+
+def myAddition(a, b):
+    return a + b
+
+
+def myMultiplication(a, b):
+    return a * b
+
+def mySubtraction(a, b):
+    return a - b
+
+def myDivision(a, b):
+    return a / b #When you are asked something, you take in the knowledge, you process the knowledge, and then you return with an answer. We can make the laptop do the same by writing return.
+    
```

Comment:
remove the unnecessary spaces and please do camelcase

Reply:
camel case